fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/JDBCConstants.java

```diff
@@ -137,7 +137,8 @@
         supportedModifiersMap.put(Type.STRING, Arrays.asList(Modifier.EXACT, Modifier.CONTAINS, Modifier.MISSING));
         supportedModifiersMap.put(Type.REFERENCE, Arrays.asList(Modifier.TYPE, Modifier.MISSING, Modifier.IDENTIFIER));
         supportedModifiersMap.put(Type.URI, Arrays.asList(Modifier.BELOW, Modifier.ABOVE, Modifier.MISSING));
-        supportedModifiersMap.put(Type.TOKEN, Arrays.asList(Modifier.MISSING, Modifier.NOT, Modifier.OF_TYPE, Modifier.IN, Modifier.NOT_IN, Modifier.TEXT));
+        supportedModifiersMap.put(Type.TOKEN, Arrays.asList(Modifier.MISSING, Modifier.NOT, Modifier.OF_TYPE,
+            Modifier.IN, Modifier.NOT_IN, Modifier.TEXT, Modifier.ABOVE, Modifier.BELOW));
```

Comment:
style suggestion: double-indent wrapped lines
```suggestion
                Modifier.IN, Modifier.NOT_IN, Modifier.TEXT, Modifier.ABOVE, Modifier.BELOW));
```

Reply:
Done.